src/theme/variables/edgeDark.js

```diff
@@ -97,6 +97,9 @@ export const edgeDark: Theme = {
   backgroundGradientLeft: palette.edgeNavy,
   backgroundGradientRight: palette.darkBlue,
 
+  // Overlay
+  overlayBackground: palette.gray,
```

Comment:
What is this? Looking at the theme, without searching the code-base for usage examples, I would have no idea. Would "cameraOverlay" make more sense?

Reply:
Still an issue. We need a better name.